endpoints.json

```diff
@@ -83,5 +83,14 @@
     "description":"DELETE the given comment by comment_id",
     "queries":[],
     "exampleResponse":{}
+  },
+  "GET /api/users":{
+    "description":"serves an array of all users",
+    "queries":[],
+    "exampleResponse":{
```

Comment:
Remember that you'll get back an array on a key of users so this needs tweaking.